lib/triggers/queryOrders.js

```
@@ -125,6 +125,14 @@ function processTrigger(msg, cfg, next, snapshot) {
         }
     }
 
+    function convertSyncInfoIntoObject(ordersResponse) {
```

Can we throw an error if there are multiple objects in the array?